plugins/network/NetEng.hlf

```diff
@@ -44,30 +44,34 @@ $ #Network browser configuration#
  Enables adding "Network" item to the disks menu.
 
  #Add to plugins menu#
- Enables adding "Network" item to the plugins menu.
+ Enables adding "Network browser" item to the plugins menu.
 
- #Выводить ".." в списке провайдеров#
- Показывать ".." на панели в списке провайдеров.
+ #Show ".." in the providers#
+ Show ".." on the panel in the list of providers. Selecting ".." will switch to the current working directory.
 
  #Show local network#
  If this option is enabled, then, when the Network Browser is first opened, it will show
 the network neighborhood of the current computer (computers in the same domain or
 workgroup). The option works only for Microsoft Windows networks.
 
- #Показывать принтеры#
- Показывать предоставляемые хостом общие принтеры.
+ #Show printers#
+ Show shared printers.
 
- #Показывать полные пути к общим ресурсам#
- ...
+ #Show share full name#
+ When the option is enabled, \\Server1 and \\Server1\Share1 are displayed, when disabled - Server1 and Share1
 
  #Show hidden shares#
- If this option is enabled, hidden (administrative) shares are shown in the plugin.
+ Опция управляет показом скрытых (административных) общих ресурсов компьютера.
 
- #Показывать скрытые ресурсы атрибутом Hidden#
- Если включена эта опция, плагин для скрытых административных ресурсов ставит атрибут Hidden.
+ - #Never show#
+   There is no search for hidden shares.
```

Comment:
*"There is no spoon"* :)
`Don't show hidden shares` or `hidden shares are never shown` or something like that might sound more natural.